db/migrate/20160919115131_add_details_to_system_consoles.rb

```diff
@@ -0,0 +1,7 @@
+class AddDetailsToSystemConsoles < ActiveRecord::Migration[5.0]
+  def change
+    add_column :system_consoles, :url, :string
+    add_column :system_consoles, :proxy_port, :integer
+    add_column :system_consoles, :proxy_status, :integer
```

Comment:
What does proxy_status as an integer mean?  Is there a mapping table?  Should it be a string so it's at least readable?